server/index.js

```diff
@@ -0,0 +1,38 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const path = require('path');
+
+const Person = require('../db/Person.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(express.static(path.join(__dirname, '../client/dist')));
+
+app.get('/people/:org', (req, res) => {
+  const company = req.params.org;
+
+  // company query is case sensitve
+    // time permitting, should have set all company names as lowercase before seeding into db
+    // then query lowercase name
+
+  Person.find({ company }).sort({ updated_at: -1 })
+    .then((result) => {
+      const members = [];
+      const team = [];
+
+      // if a person updated their title multiple times,
+      // there are duplicate entries of that person, and we only want their most recent update
+      result.forEach((row) => {
+        if (!members.includes(row.member_id)) {
```

Comment:
Hi Rosarlin, I believe mongodb automatically checks for unique '_id' so I am thinking you don't have to check for this (it will give an error).

Reply:
That makes sense, seems I didn't look through the code thoroughly enough.

FYI there's a mono aggregate method that can do this for you which is a little involved but could be worth looking into sometime (especially if you want to offload calculation to the database server).

Reply:
On a second look through, you could probably get rid of body parser if you are not expecting to handle any post, patch, etc. Otherwise, looks great.

Reply:
Thanks for the suggestion! I will look into it. I agree that it's better for the database server to handle this calculation.